rich/logging.py

```diff
@@ -89,18 +104,26 @@ def emit(self, record: LogRecord) -> None:
         if self.KEYWORDS:
             message_text.highlight_words(self.KEYWORDS, "logging.keyword")
 
-        self.console.print(
-            self._log_render(
-                self.console,
-                [message_text],
-                log_time=log_time,
-                time_format=time_format,
-                level=level,
-                path=path,
-                line_no=record.lineno,
-                link_path=record.pathname if self.enable_link_path else None,
+        if self.handle_tracebacks and record.exc_info:
```

Comment:
This will render the exception without the other configured columns (time, severity, filename). Logged exceptions also have a message which I don't think we're displaying here.

You can capture the current exception with `rich.exception.Exception` then add that to the log renderer. i.e. if there is an exception, call `_log_render` with `[message_text, my_exception]`